src/Android/Properties/AndroidManifest.xml

```diff
@@ -1,57 +1,48 @@
+	<uses-permission android:name="android.permission.CAMERA" />
+	<uses-permission android:name="android.permission.USE_FINGERPRINT" />
+	<uses-permission android:name="android.permission.USE_BIOMETRIC" />
+	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+	<uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
+	<uses-permission android:name="com.samsung.android.providers.context.permission.WRITE_USE_APP_FEATURE_SURVEY" />
+	<uses-feature android:name="android.hardware.camera" android:required="false" />
+	<uses-feature android:name="android.hardware.camera.autofocus" android:required="false" />
+	<application android:label="Bitwarden" android:theme="@style/LaunchTheme" android:allowBackup="false" tools:replace="android:allowBackup" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:networkSecurityConfig="@xml/network_security_config">
+		<provider android:name="androidx.core.content.FileProvider" android:authorities="com.x8bit.bitwarden.fileprovider" android:exported="false" android:grantUriPermissions="true">
+			<meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/filepaths" />
+		</provider>
+		<meta-data android:name="android.max_aspect" android:value="2.1" />
+		<meta-data android:name="android.content.APP_RESTRICTIONS" android:resource="@xml/app_restrictions" />
+		<!-- Support for Samsung "Multi Window" mode (for Android < 7.0 users) -->
+		<meta-data android:name="com.samsung.android.sdk.multiwindow.enable" android:value="true" />
+		<meta-data android:name="com.samsung.android.sdk.multiwindow.penwindow.enable" android:value="true" />
+		<!-- Support for LG "Dual Window" mode (for Android < 7.0 users) -->
+		<meta-data android:name="com.lge.support.SPLIT_WINDOW" android:value="true" />
+		<!-- Declare MainActivity manually so we can set LaunchMode using API dependant resource -->
+		<activity android:name="com.x8bit.bitwarden.MainActivity" android:configChanges="keyboard|keyboardHidden|navigation|orientation|screenSize|uiMode" android:exported="true" android:icon="@mipmap/ic_launcher" android:label="Bitwarden" android:launchMode="@integer/launchModeAPIlevel" android:theme="@style/LaunchTheme">
+			<intent-filter>
+				<action android:name="android.intent.action.MAIN" />
+				<category android:name="android.intent.category.LAUNCHER" />
+			</intent-filter>
+			<intent-filter>
+				<action android:name="android.intent.action.SEND" />
+				<category android:name="android.intent.category.DEFAULT" />
+				<data android:mimeType="application/*" />
+				<data android:mimeType="image/*" />
+				<data android:mimeType="video/*" />
+				<data android:mimeType="text/*" />
+			</intent-filter>
+		</activity>
+	</application>
+	<!-- Package visibility (for Android 11+) -->
+	<queries>
+		<intent>
+			<action android:name="*" />
+		</intent>
+	</queries>
+</manifest>
```

Comment:
Will the removal of the trailing newline make dotnet format angry?

Reply:
@mp-bw I've checked with `dotnet tool run dotnet-format --check` and it passes

Reply:
@fedemkr Cool.  One last thing to check: When you edit the file in VS Win, does the auto formatting re-insert the newline?

Reply:
@mp-bw not sure, but I think that nobody is using VS Win for mobile. I don't have an updated VS Win running here but I could set that up this week.

Reply:
@fedemkr You may be right - It was more common when I first joined.  Don't worry about testing it, we'll deal with it if/when it happens.